.cirrus.yml

```diff
@@ -278,12 +278,54 @@ task:
         # Empirically, as of October 2019, the docs-linux shard took about 30 minutes when run with
         # 1 CPU and 4G of RAM. 2 CPUs reduced that to 20 minutes, more CPUs did not improve matters.
         CPU: 2
+      docs_cache:
+        folder: dev/docs
+        fingerprint_script:
+          - echo "docs_${CIRRUS_CHANGE_IN_REPO}_${CIRRUS_BUILD_ID}"
+      script:
+        - ./dev/bots/docs.sh docs
```

Comment:
I'd put the creation of the cache in `populate_script` of the cache instruction since cache might fail to be downloaded or have some other issue.